### ISSUE TITLE
Use lighter neural networks for unit test cases

### ISSUE THREAD

**Issue by myungjoo** [nnstreamer/nnstreamer#2169]
We don't need accurate neural networks.
Often, we don't even need any "sound" neural networks.
Sometimes you just need to test if you can feed inputs and get outputs from the networks (e.g., #2155).

Thus, in many cases neural networks that do "Output[i,j,k] = Input[i,j,k] + Input[k.j.i]" (or anything similarly stupid) should be just fine. Let's not use really working neural networks for unit testing. They are already taking too much space.

In other words, let's use OO kiB test models instead of 1X MiB real models.
(Replace larger models with smaller models in /tests/test_models/* that can still test
 all test cases and require a few operations in the given frameworks)

**Comment by wooksong** [comment]
Acked and agreed

**Comment by myungjoo** [comment]
But be careful, it is known that the memory-handling behavior of tf-lite changes when the model size of input/output size is small.

**Comment by myungjoo** [comment]
Looks like smaller models have side-effects of hiding a few possible regressions. Let's revisit this if we really need to down-size things.
